.azuredevops/pipelineVariables/global.variables.yml

```diff
@@ -0,0 +1,84 @@
+variables:
```

Comment:
we need to create an issue for the documentation process for how to get started with ADO pipelines and what others need to modify

Reply:
An issue for the Wiki exists. I'll try to cover it all there